CHANGELOG.md

```diff
@@ -1,285 +1,2 @@
-
-## [0.8.4](https://github.com/os97673/debase-ruby_core_source/compare/v0.8.3...v0.8.4)
-
-* ruby 2.3.0 added
-
-## [0.8.3](https://github.com/os97673/debase-ruby_core_source/compare/v0.8.2...v0.8.3)
-
-* ruby 2.0.0-p648 added
-* ruby 2.1.8 added
-* ruby 2.2.4 added
-
-## [0.8.2](https://github.com/os97673/debase-ruby_core_source/compare/v0.8.1...v0.8.2)
-
-* ruby 2.3.0-preview2 added
-
-## [0.8.1](https://github.com/os97673/debase-ruby_core_source/compare/v0.8.0...v0.8.1)
-
-* ruby 2.3.0-preview1 added
-
-## [0.8.0](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.10...v0.8.0)
-
-* previous version of ruby is used if there is no headers for the
-  current one.
-
-## [0.7.10](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.9...v0.7.10)
-
-* ruby 2.2.3 added
-* ruby 2.1.7 added
-* ruby 2.0.0-p647 added
-
-## [0.7.9](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.8...v0.7.9)
-
-* ruby 2.0.0-p645 added
-* ruby 2.1.6 added
-* ruby 2.2.2 added
-
-## [0.7.8](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.7...v0.7.8)
-
-* ruby 2.0.0-p643 added
-
-## [0.7.7](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.6...v0.7.7)
-
-* added missing ruby 2.2.1 patchlevel
-
-## [0.7.6](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.5...v0.7.6)
-
-* ruby 2.2.1 added
-
-## [0.7.5](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.4...v0.7.5)
-
-* ruby 2.2.0 added
-
-## [0.7.4](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.3...v0.7.4)
-
-* ruby 2.2.0-rc1 added
-
-## [0.7.3](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.2...v0.7.3)
-
-* ruby 2.2.0-preview2 added
-
-## [0.7.2](https://github.com/os97673/debase-ruby_core_source/compare/v0.7.0...v0.7.2)
-
-* sources for ruby 2.0.0-p598 added
-* sources for ruby 2.1.5 added
-
-## [0.7.0](https://github.com/os97673/debase-ruby_core_source/compare/v0.6.0...v0.7.0)
-
-* ruby 2.1.4 added
-* ruby 2.0.0-p594 added
-* ruby 1.9.3-p550 added
-
-## [0.6.0](https://github.com/os97673/debase-ruby_core_source/compare/v0.5.0...v0.6.0)
-
-* fixed headers for 2.2.0-preview1
-
-## [0.5.0](https://github.com/os97673/debase-ruby_core_source/compare/v0.4.0...v0.5.0)
-
-* sources for ruby 2.0.0-p576 added
-
-## [0.4.0](https://github.com/os97673/debase-ruby_core_source/compare/v0.3.0...v0.4.0)
-
-* build fixed
-
-## [0.3.0](https://github.com/os97673/debase-ruby_core_source/compare/v0.2.0...v0.3.0)
-
-* gem installation fixed
-
-## [0.2.0](https://github.com/os97673/debase-ruby_core_source/compare/v0.1.0...v0.2.0)
-
-* gem's homepage fixed
-* gem required ruby ~> 2.0
-* sources for < 2.0 removed
-
-## 0.1.0
-
-* Initial release - gem renamed to debase-ruby_source_code, Debugger module renamed to Debase
-* sources for 2.1.3 and 2.2.0-preview1 added
-
+* Initial release - gem renamed to datadog-ruby_source_code, Debase module renamed to Datadog
+* only sources for 2.5, 3.3, and 3.4 included
```

Comment:
This will be an endless source of conflicts if the plan is to merge upstream changes. How about leaving this CHANGELOG.md unchanged, adding a new file called e.g. CHANGELOG.datadog.md with your new contents and any further changes that we make (including the points at which we update this fork from upstream). This way the original changelog won't conflict, we retain the upstream changes, and we have a place to put our changes.